src/framework/Framework_Snapshot_001.test.js

```diff
@@ -0,0 +1,46 @@
+import {
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+  screenshot,
+  webKitBrowserOps,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'Framework snapshot test',
+  description: 'Tests if the Framework snapshot module works',
+  teardown: restartFramework,
```

Comment:
No need

Reply:
Remove this code related to restartFramework